### PR TITLE
New plugin to set alt text on image

### PR DESCRIPTION
**Pull request by Misticek**

### CODE REVIEW

File: src/modules/AltAttribute.js [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+    }
+    input.addEventListener('input', (e) => {
+      this.img.alt = e.target.value
+    })
+
+    this.display.insertAdjacentHTML('beforeend', 'Alt: ')
+    this.display.appendChild(input)
+  }
+
+  onDestroy = () => {}
```
Comment (capaj) [inline src/modules/AltAttribute.js]: ARE you sure we don't need to clean up those event listeners here @Misticek ?
Seems like that could be a little memory leak, unless the input is removed from the DOM on destroy by something else obviously.

**Review by capaj — APPROVED** [review]
LGTM, will test on our staging lol

File: src/modules/AltAttribute.js (hunk 1 above)
Comment (capaj) [reply src/modules/AltAttribute.js]: you might want to store your DOM node on `this`
ideally as `this.altInput` or something like that

Comment (Misticek) [reply src/modules/AltAttribute.js]: yep I know, I accidentally named it same as you suggested 😄

**Review by cztomsik — APPROVED** [review]

**Pull request merged**